specs/fiscal-data-package.md

```diff
@@ -36,517 +37,1746 @@ body:
+This document is a work-in-progress towards Fiscal Data Package v1.
 
-The specification is both simple and easy for publishers to use and, at the same time, sufficiently rich and structured to be useful and processable - especially machine processable - by consumers. In particular, Fiscal Data Packages are:
+For more general notes and thoughts that lead into this document, [see here](https://hackmd.io/MYNgrCBGIBwGYFoCGcAsSGtWAjAgnGAAx5EBMAJmegOxjZEDMQA=?both).
 
-* Made from lightweight and easily made components (CSV data, JSON metadata)
-* Structured according to a simple open standard
-* Self-documented with metadata
-* Including sufficient information to allow for automated and standardized processing and analysis
+For the public issues tracking work towards FDP v1, [see here](https://github.com/frictionlessdata/specs/labels/Fiscal%20Data%20Package).
 
-Fiscal Data Package specifies the *form* for fiscal data and offers a standardized framework for the *content*. By giving a common *form* to budget data, Fiscal Data Package frees data users from the artificial obstacles created by the lack of a standard structure. By clarifying the *content* of budget data and recommending standard information that fiscal data should contain, Fiscal Data Package helps make data releases more comparable and useful. The Fiscal Data Package specification provides support for:
+ ### What is Fiscal Data?
 
-* A simple and standard way to provide rich metadata about fiscal information - where it came from, who produced it, how it is licensed, what time period it covers etc
-* Mapping the raw "physical" model, as represented by columns in the data files, to a standardized "logical" model based around basic fiscal concepts: amounts spent, suppliers, administrative and functional classifications etc
-* Progressive enhancement of data via a range of *recommended*, but not *required* metadata, in order to establish a clear path for data providers to enhance data quality, and to address new use cases going forward.
+In the context of this specification, we will define a fiscal dataset, at its core, to be one providing information on a series of fiscal transactions.
 
-Fiscal Data Package builds on the [Data Packages specifications][dp]. It defines a "profile" that adds some additional constraints.  It also extends the [Tabular Data Package][tdp] profile (`tabular`) which itself extends the `base` Data Package format.  Thus, any Fiscal Data Package is also a [Tabular Data Package][tdp], and is also a [Data Package][dp].
+Each of these transactions consists of the following concepts:
+- The *source* of the transaction
+- The *recipient* of the transaction
+- *When* the transaction occurred
```

Comment:
This feels awkward for talking about transactions that represent budgets. 

A date might represent when the budget was confirmed - but it's quite abstract to think of this as a point of 'occurrence'